### ISSUE TITLE
Redefine Brahmarandhra (cranial hole)

### ISSUE THREAD

**Issue by cristiano-moscatelli** [DiGArtefacts/thesaurus#122]
@jessiepons @SerenaAutiero 

The term brahmarandhra is too specific and we have to adopt a more neutral definition. 
"Cranial hole" doesn't work either since bodhisattvas can have it as well. 

What about just "hole"?

**Comment by jessiepons** [comment]
I don't follow why "cranial hole" does not work for bodhisattvas. Can you please explain?

**Comment by cristiano-moscatelli** [comment]
Bodhisattvas don't have the usnisa

**Comment by jessiepons** [comment]
indeed they don't. I guess referring to cranial hole could imply that their bun (in which the hole is) is in fact a cranial protuberance. Hence the inadequacy, right?

**Comment by cristiano-moscatelli** [comment]
Yes! If my memory serves me, Klimburg-Salter & Taddei describe that feature on bodhisattva sculptures just as “hole in the head”.

We could adopt top-hole as generic concept. What do you think?

**Comment by jessiepons** [comment]
And apply it to both Buddhas and bodhisattvas? Do you think using "hole in the head" would be too ambiguous? "Top-hole" is fine, just not super elegant. But really, I will not fight for or against it.

**Comment by cristiano-moscatelli** [comment]
I’m fine with hole in the head, I don’t think it’s ambiguous. And yes, I would apply it to both buddhas and bodhisattva. 

Let’s see if Serena has any suggestions!
